docs/software/file_servers.md

```diff
@@ -2,13 +2,171 @@
 
 ## Overview
 
-!!! warning "Documentation in progress..."
+- [**ProFTPD - Simple, efficient, lightweight FTP file server**](#proftpd-simple-efficient-lightweight-ftp-file-server)  
+- [**Samba - Feature rich file server**](#samba-feature-rich-file-server)  
+- [**vsftpd - Feature rich FTP file server**](#vsftpd-feature-rich-ftp-file-server)  
+- [**NFS - Network file system server**](#nfs-network-file-system-server)  
 
-    The links redirect to the old documentation available on [DietPi.com Forum](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5). **We need your help to create and update this documentation ! Thank you** :octicons-heart-16: :octicons-heart-16:
+??? info "How do I run **DietPi-Software** and install **Optimised software** ?"
+    To install any of the **DietPi Optimised Software** listed below run from the command line:
+    ```
+    dietpi-launcher
+    ```
+    and selected _DietPi-Software_. or launch directly the tool:
+    ```
+    dietpi-software
+    ```
 
-- [ProFTPD - Simple, efficient, lightweight FTP file server](https://dietpi.com/phpbb/viewtopic.php?p=55#p55)
-- [Samba - Feature rich file server](https://dietpi.com/phpbb/viewtopic.php?p=56#p56)
-- [vsFTPD - Feature rich FTP file server](https://dietpi.com/phpbb/viewtopic.php?p=2820#p2820)
-- [NFS - Network file system server](https://dietpi.com/phpbb/viewtopic.php?p=2821#p2821)
+    Choose **Software optimised** and select one or more items. Finally click on `Install`. DietPi will do all the necessary steps to install and start these software items.
+
+    ![DietPi software](../assets/images/dietpi-software.jpg)
+
+    To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools) section.
 
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
+
+## ProFTPD - Simple, efficient, lightweight FTP file server
+
+ProFTPD gives you access to files/music/downloads etc. on your DietPi system quickly and efficiently with minimal overhead.
+
+![DietPi file server software ProFTPD](../assets/images/dietpi-software-fileservers-proftpd.png)
+
+### Access to ProFTPD
+
+The access to ProFTPD with **Windows** is achieved as follows:
+
+- Go to My Computer (Windows Explorer).
+- In the address bar at the top type `ftp://dietpi:dietpi@192.168.0.100` press enter.
```

Comment:
The second "dietpi" is the global software password, which should be changed. I generally wanted to replace all cases where we either name the global software password of the local IP address. On MkDocs we can do that very nicely:
1. Enable Markdown abbr extension and make `yourGlobalSoftwarePassword` (or a shorter version) an abbreviation where the long form simply explains it a bid longer.
2. Do the same with `192.168.0.100`.
3. Use those two strings consequently.
4. Alternatively make them not abbreviations but links to a dedicated page, where the password, how it is changed, respectively the local IP and how to check it is explained, or even both: link and abbraviation:
![test](https://user-images.githubusercontent.com/28480705/96899641-71fab000-1491-11eb-89b7-ef6a627c98ef.png)
As can be seen, it can be both, even inside code and pre blocks, although the links are not checked by our GitHub action when in those blocks. Only a link inside the abbreviation popup does not work 😄.

Reply:
@MichaIng: Would you please put this on the todo-list for "#### Decision?"